### DISCUSSION TITLE
Prepared images for some popular regions.

### DISCUSSION THREAD

**Discussion by Apo-S** [mediagis/nominatim-docker#448] [Ideas]
I think it would be great to have ready-made images for some popular regions.

When I just downloaded the image / launched it and everything works right away.

This is what docker can provide.

Of course, the various database parameters, other parameters and the data themselves are different for everyone.

But I would just like to have something that you can run and try.

I launched on my machine where windows with wsl2 us-south-latest on hdd 5400rpm, i5-10400, with RAM limits ~16 times less than the standard ones and I don’t understand when the first launch will be completed and whether it will be possible to wait for this, already the third day.

**Comment by mtmail** [comment]
Can you post the last lines of output? Then it should be possible to estimate how long the import will take.

Have you tried a smaller region first? I tend to use the countries Monaco or Liechtenstein which on a 2core,4GB RAM,SSD take less than 30min to import. https://download.bbbike.org/osm/ has some city extracts (look for *.pbf files).

HDD 5400rpm is bad news: Nominatim at its core is a database with huge indices and does a lot of random access. The default database performance settings in the Nominatim installation are for SSD/NVMe drives. With HDD you can look at weeks of import time (depending on region, RAM/CPU).

**Reply by Apo-S** [reply]
I did not try other regions, I started the test with us-south. I'll take a look at bbbike, thanks.

last line:
`2023-05-15 11:26:39: Done 10193020 in 71695 @ 142.170 per second - rank 30 ETA (seconds): 2398.96`

**Reply by leonardehrenfried** [reply]
142 is very slow. With a SSD you can expect a 10x speed improvement.

**Reply by mtmail** [reply]
Your import is about 70% done. After rank 30 follows rank 0, then interpolation lines, then postcodes.

**Reply by Apo-S** [reply]
thanks, so I have ~26 hours left, I'll wait
